src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/geo/GEOMiner.java

```diff
@@ -274,7 +270,7 @@ public boolean onClick(Player p, int slot, ItemStack cursor, ClickAction action)
 
                 @Override
                 public boolean onClick(InventoryClickEvent e, Player p, int slot, ItemStack cursor, ClickAction action) {
-                    return cursor == null || cursor.getType() == null || cursor.getType() == Material.AIR;
```

Comment:
I know that this is likely marked as Nonnull/NotNull yet it is possible for this to return null. It's rare and super annoying due to the warnings this raises in the IDE, but please revert this line.

Reply:
Ah I see, you are correct. Reverted.